packages/calypso-polyfills/index.node.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Internal dependencies
+ */
+require( './node' );
```

Comment:
The `index.node.js` and `index.web.js` files are a bit redundant. You could directly specify the target files in `package.json`:
```
  "main": "node.js",
  "browser": "browser-fallback.js"
```
Using names `browser-evergreen.js` and `browser-fallback.js` (i.e., with prefix) is also a good idea in my opinion.

Reply:
Good ideas! Done!